### PR TITLE
docs: Update news with move from CLA to DCO

### PR DESCRIPTION
**Pull request by edlee2121**
Signed-off-by: Edward Lee <edward_lee@intuit.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,13 @@
 
 ## News
 
+Argo Project is moving from CLA to DCO as a more contributor friendly way to track contributions.
+* https://github.com/apps/dco/
+Please start signing off your contributions by doing ONE of the following:
+* Run `git config commit.gpgsign true` in your argo repo to automatically signoff future commits.
+* Use `git commit -s ...` with each commit to add signoff.
```
Comment (alexec) [inline README.md:13]: `error: cannot run gpg: No such file or directory`

File: README.md [hunk 2]
```diff
@@ -6,6 +6,13 @@
 
 ## News
 
+Argo Project is moving from CLA to DCO as a more contributor friendly way to track contributions.
```
Comment (alexec) [inline README.md:9]: how is this more friendly?

File: README.md (hunk 1 above)
Comment (alexec) [reply README.md:13]: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/changing-access-permissions-for-wikis

Comment (edlee2121) [reply README.md:13]: Alex, you don't need gpg. You should disable that.

File: README.md (hunk 2 above)
Comment (edlee2121) [reply README.md:9]: The DCO is much shorter to read and understand.

File: README.md (hunk 1 above)
Comment (alexec) [reply README.md:13]: when I followed those instructions, it worked

File: README.md [hunk 3]
```diff
@@ -6,6 +6,13 @@
...
+* https://github.com/apps/dco/
+Please start signing off your contributions by doing ONE of the following:
+* Run `git config commit.gpgsign true` in your argo repo to automatically signoff future commits.
```
Comment (alexec) [inline README.md:12]: this did not work for me, I think I have an older Git which does not have this setting

Comment (edlee2121) [reply README.md:12]: Whoops, I need to delete the gpgsign part.

Comment (edlee2121) [reply README.md]: I deleted the gpgsign part.

**Comment by edlee2121** [comment]
@alexec , are you good with the PR?

**Comment by alexec** [comment]
It is fine. It is a bit more work for developers, as they must sign all commits, but no one has complained.

Can I ask you disable CLA at some point?

**Comment by edlee2121** [comment]
Thanks. When we switch over in about a week or two, CLA will be disabled.

**Review by alexec — APPROVED** [review]

**Pull request merged**